packages/react-components/src/components/Dialpad/Dialpad.test.tsx

```diff
@@ -33,39 +42,33 @@ describe('Dialpad tests', () => {
     const testLocale = createTestLocale({
       dialpad: dialpadStrings
     });
-    const component = mountWithLocalization(<Dialpad />, testLocale);
-    expect(component.find('[data-test-id="dialpad-input"]').first().props().placeholder).toBe(
-      testLocale.strings.dialpad.placeholderText
-    );
+    renderWithLocalization(<Dialpad />, testLocale);
+    expect(screen.getByRole('textbox').getAttribute('placeholder')).toBe(testLocale.strings.dialpad.placeholderText);
```

Comment:
is `screen` a property that just exists inside the test runner we can query?

Reply:
Yeah, its basically a way of saying - hey query what's on the "screen" (screen isn't a real screen, just an in memory interpretation of a screen). Ideally we'd only use screen for queries but it can't query anything arbitrary (like data-ui-id) so for those I've used the `container` returned by the render method